src/components/molecules/ScheduleEvent/ScheduleEvent.tsx

```diff
@@ -39,12 +54,36 @@ export const ScheduleEvent: React.FC<ScheduleEventProps> = ({
   scheduleStartHour,
   personalizedEvent: isPersonalizedEvent = false,
 }) => {
+  const eventRef: RefObject<HTMLDivElement> = useRef(null);
+  const isMinimallyLongEvent = event.duration_minutes >= MIN_LONG_EVENT_LENGTH;
+  const isShortEvent = event.duration_minutes <= SHORT_EVENT_LENGTH;
+  const isEventLong = event.duration_minutes >= LONG_EVENT_LENGTH;
+
```

Comment:
Given these don't seem to be used until the other `const`s that I just said to move above `containerClasses`, it would make much more sense for them to be just before those. They are all 'related' so keep them together.

Reply:
done f718762